catalyst-gateway/bin/src/event_db/queries/event/proposal.rs

```diff
@@ -60,7 +60,7 @@ impl ProposalQueries for EventDB {
             .query(Self::PROPOSAL_QUERY, &[&event.0, &objective.0, &proposal.0])
             .await?;
         let row = rows
-            .get(0)
+            .first()
             .ok_or_else(|| Error::NotFound("cat not find proposal value".to_string()))?;
```

Comment:
I know that this wasn't introduced in this PR, but seems like a good time to fix it. There's a typo in the error, should read "can/could not find proposal value" or something to that effect.

Reply:
Didn't see that, will fix it